Sources/BugsnagPerformance/Private/Instrumentation/NetworkInstrumentation.mm

```diff
@@ -72,6 +72,12 @@ - (void)URLSession:(__unused NSURLSession *)session task:(NSURLSessionTask *)tas
         return;
     }
 
+    auto httpResponse = BSGDynamicCast<NSHTTPURLResponse>(task.response);
```

Comment:
Can we get a test [like this](https://github.com/bugsnag/bugsnag-android-performance/pull/123/files#diff-2b40cecec95e29885ecaa8716245aa571bd5f31de25486bb7ec9a21099dbf831R21-R24) setup?

Reply:
Added!